metadata-ingestion/src/datahub/ingestion/source/tableau.py

```diff
@@ -2462,35 +2437,6 @@ def emit_embedded_datasources(self) -> Iterable[MetadataWorkUnit]:
                 is_embedded_ds=True,
             )
 
-    @lru_cache(maxsize=None)
```

Comment:
I lose the lru_cache capabilities, but I don't think that matters... these are such simple and quick operations